src/components/toolbar/search-component.ts

```diff
@@ -81,20 +118,30 @@ export function view(
         ? html.div(
             {
               class: 'result-container',
+              scrollTop: (_state.focusedEl && _state.keyPress) ?
+                            _state.focusedEl * 60 : 0,
             },
-            _state.results.map((result: any) => {
+            _state.results.map((result: any, i: number) => {
               const value = result.matches[0].value
               const formattedValue = value.length > 256
                 ? value.substring(0, 256) + '...'
                 : value
               return html.div(
                 {
-                  class: 'search-result',
+                  class: (_state.focusedEl === i) ?
```

Comment:
If you just want to apply additional style on top of an existing style you can set multiple classes, `a b`. Then all the style of `a` will be applied and any additional styles specified in `b` will be applied on top. I suggest you look at the `classNames` helper function to make it easier (https://github.com/JedWatson/classnames#usage)